### PR TITLE
Various Edits (mosty Fedimian)

### PR DESCRIPTION
**Pull request by vtange**
Fedimian -> Holy District and Market District
some missed Fedimian dialogue
great cathedral -> Great Cathedral
cursed city -> Cursed City
Crossfire attribute proper translation

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -7960,7 +7960,7 @@ QUEST_LV_0100_20150908_007955	Putting the oil
 QUEST_LV_0100_20150908_007956	The magic supplier has been fixed{nl}Activate the secondary purifier
 QUEST_LV_0100_20150908_007957	The secondary purifier is working again
 QUEST_LV_0100_20150908_007958	It seems that a part of the purifier is missing{nl}Use the mine compass to look for the part
-QUEST_LV_0100_20150908_007959	The mine compass is pointing at 6th districte{nl}Look for the parts at 6th district
+QUEST_LV_0100_20150908_007959	The mine compass is pointing at 6th District{nl}Look for the parts at 6th District
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:7963]: 6th District -> District 6?

Comment (vtange) [inline QUEST_LV_0100.tsv:7963]: the whole mine looks like it needs the same standardization work I did for Mausoleum and Mage Tower. this is just a spell check.
do you want to do the mines? or should I take a break from the high level areas and go do the mines/tenet?

Comment (Sunaries) [inline QUEST_LV_0100.tsv:7963]: I'm willing to do it, but I'm first checking the upcoming areas. Though, I think the Crystal Mine looks pretty good so far.

**Comment by vtange** [comment]
Good to merge

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**